pkg/mermaid/datamodeldiagram/datamodeldiagram.go

```diff
@@ -10,6 +10,10 @@ import (
 
 //cause linter..
 const ref = "ref"
+const enum = "enum"
+const mapS = "map"
+const relation = "relation"
+const tuple = "tuple"
```

Comment:
These could be declared in a const block, might be a bit cleaner

const (
   ref         = "ref"
   enum    = "enum"
   mapS    = "map"
   relation = "relation"
   tuple     = "tuple"
)

Reply:
Addressed!